pkg/utils/transform/transform.go

```diff
@@ -65,11 +65,44 @@ func InjectNamespaceConditional(preserveNamespace, targetNamespace string) mf.Tr
 		if ok && val == "true" {
 			return nil
 		}
-
 		return tf(u)
 	}
 }
 
+func InjectNamespaceRoleBinding(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		switch strings.ToLower(u.GetKind()) {
```

Comment:
nit: this can be better written as 

```go

return func (u ...) error {
  kind := strings.ToLower(u.GetKind());   
  if kind != "rolebindings" {
    return nil  
  }
subjects, _, _ := unstructured.NestedField...
return nil 
}
```